ui/src/main/scripts/plugins/directives/instance-view-diagram/ts/instanceViewDiagramDirective.ts

```diff
@@ -198,7 +198,7 @@ module InstanceViewDiagram {
           _.each(d.properties, (property) => {
             if (!!property.value) {
               nodeTT += ('<li class=\'tt-prop\'><strong>' + property.name + '</strong> ' + property.value + '</li>');
-              if (property.name === 'fault') {
+              if (property.name === 'fault' || property.name === 'error') {
```

Comment:
Sorry we should have been more precise, we need to check for property 'error' with value 'true'. Its possible that error=false may be specified. You may need to deal with different cases.